Sources/iYuga/Classifier.swift

```diff
@@ -20,14 +20,15 @@ public class Classifier {
     
     public init() {}
     
-    public func getYugaTokens(_ sentence: String) -> Pair<String, Dictionary<String, AnyObject>> {
+    public func getYugaTokens(_ message: String) -> Pair<String, Dictionary<String, AnyObject>> {
         var configMap = Dictionary<String, String>()
         configMap[Constants.YUGA_CONF_DATE] = Constants.dateTimeFormatter().string(from: Date(milliseconds: 1527811200000))
-        return getYugaTokens(sentence, configMap, IndexTrack(next: 0))
+        return getYugaTokens(message + " ", configMap, IndexTrack(next: 0))
```

Comment:
Isn't it better to handle not having space in `getYugaTokens` than adding space here? Doesn't it mean everyone who wants to call `getYugaTokens` have to remember this?

Reply:
I have removed the space as it was not needed.